internal/bminventory/inventory.go

```diff
@@ -3715,15 +3715,15 @@ func (b *bareMetalInventory) generateV2NextStepRunnerCommand(ctx context.Context
 			params.NewHostParams.HostID.String(), params.InfraEnvID.String(), params.NewHostParams.DiscoveryAgentVersion, b.AgentDockerImg)
 	}
 
-	config := hostcommands.V2NextStepRunnerConfig{
+	config := hostcommands.NextStepRunnerConfig{
```

Comment:
Why did you remove the v2? v1 will be deprecated and removed

Reply:
This is v2 , i just changed the name to the normal one. There is only one file now so no need to call it v2 or any other version

Reply:
I'm not sure if the agent will know how to parser it. isn't it a different struct?

Reply:
agent will not get this struct as it is internal one, agent will get next_step struct

Reply:
this is internal struct we can call it what we want, it is not relevant to api itself, all the change that was done for v2 was adding infraenv-id and not cluster.

Reply:
OK, thanks for the explanation